distribution/docs/src/main/resources/content/_integrating/_endpoints/opensearch-endpoint.adoc

```diff
@@ -77,45 +64,63 @@ See <<{architecture-prefix}available_query_response_transformers,Query Response
 default: `atom`
 |===
 
-.Temporal Extension
+.Sample OpenSearch Textual Query
+[source,https]
+----
+https://<FQDN>:<PORT>/services/catalog/query?q="Aliquam"&count=20
+----
+
+==== OpenSearch Temporal Queries
+Queries can also specify a start and end time to narrow results.
+
+.OpenSearch Temporal Parameters
 [cols="4*", options="header"]
 |===
-|OS Element
-|HTTP Parameter
+|OpenSearch Element
+|HTTPS Parameter
 |Possible Values
 |Comments
 
 |`start`
 |`dtstart`
-|RFC-3399-defined value (e.g. `YYYY-MM-DDTHH:mm:ssZ`, `yyyy-MM-dd'T'HH:mm:ss.SSSZZ`)
+|RFC-3399-defined value:`YYYY-MM-DDTHH:mm:ssZ` or `yyyy-MM-dd'T'HH:mm:ss.SSSZZ`
 |Specifies the beginning of the time slice of the search.
 
 Default value of "1970-01-01T00:00:00Z" is used when `dtend` is specified but `dtstart` is not specified.
 
 |`end`
 |`dtend`
-|RFC-3399-defined value (e.g. `YYYY-MM-DDTHH:mm:ssZ`, `yyyy-MM-dd'T'HH:mm:ss.SSSZZ`)
+|RFC-3399-defined value:`YYYY-MM-DDTHH:mm:ssZ` or `yyyy-MM-dd'T'HH:mm:ss.SSSZZ`
 |Specifies the ending of the time slice of the search
 
 Current GMT date/time is used when `dtstart` is specified but `dtend` is not specified.
 |===
 
+.OpenSearch Temporal Query Example
+[source,https]
+----
+https://<FQDN>:<PORT>/services/catalog/query?q-'*'&dtstart=${timestamp}T00:00:00Z&dtend=${timestamp}T18:00:00Z
```

Comment:
❓ Should the start of the query parameters be `query?q='*'...` instead of `query?q-'*'...`?

Reply:
yes. fixed